contracts/netmap/contract.go

```diff
@@ -790,6 +810,18 @@ func UnusedCandidate() Candidate {
 	return Candidate{}
 }
 
+// GetEpochBlock returns block index when given epoch came. Returns 0 if the
+// epoch is missing.
+//
+// Use [LastEpochBlock] if you are interested in the current epoch.
+func GetEpochBlock(epoch int) int {
```

Comment:
GetEpochTime as well?

Reply:
will add. Can `LastEpochTime` be also useful?

Reply:
I think so. In fact, I'd really like to switch epochs from blocks to time. It can be done and we'd need this time data then.

Reply:
At the moment let's just expose everything we have from the contract and then we'll figure out how to use or abuse it.

Reply:
added them both